ard/My-List-Util/lib/My/List/Util.pm

```diff
@@ -0,0 +1,156 @@
+package My::List::Util;
+
+use 5.006;
+use strict;
+use warnings;
+
+=head1 NAME
+
+My::List::Util - The great new My::List::Util!
+
+=head1 VERSION
+
+Version 0.01
+
+=cut
+
+our $VERSION = '0.01';
+
+
+=head1 SYNOPSIS
+
+Provides methods for summing and shuffling lists.
+
+=head1 EXPORT
+
+=over 1
+
+=item sum()
+
+=item shuffle()
+
+=back
+
+=head1 SUBROUTINES/METHODS
+
+=head2 sum
+
+Returns the sum of the numerics in the provided list
+
+=cut
+
+sub sum {
+  my $sum;
+  foreach my $num ( grep( /^-?\d*\.?\d*$/, @_ ) ) {
```

Comment:
Perl stringifies huge numbers using scientific notation, which this will throw away.